### PR TITLE
Update docs to reflect flag change in client build scripts

### PR DESCRIPTION
**Pull request by drewgillies**
@joelverhagen @alpaix @emgarten

### CODE REVIEW

**Comment by drewgillies** [comment]
@harikmenon

File: README.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ A [full list of all the repos](https://github.com/NuGet) is available as well.
 ######In case you have build issues please clean the local repo using `git clean -xdf` and retry building
 
 ####Notable .\build.ps1 switches
-- `-SkipDev14` - builds only binaries targeting Visual Studio 15 (Dev15)
-- `-SkipDev15` - builds only binaries targeting Visual Studio 2015 (Dev14)
+- `-SkipVS14` - skips building binaries targeting Visual Studio 14 (released 2015)
+- `-SkipVS15` - skips building binaries targeting Visual Studio 15
```
Comment (alpaix) [inline README.md]: I guess it should be **"15"** with quotes

Comment (drewgillies) [inline README.md]: Have adjusted this @alpaix --thanks. Better.

**Comment by harikmenon** [comment]
lgtm

**Comment by joelverhagen** [comment]
:shipit:

**Pull request merged**